tests/test_posttest.py

```diff
@@ -30,3 +32,31 @@ def test_collect_techsupport(duthost):
         duthost.fetch(src=tar_file, dest=TECHSUPPORT_SAVE_PATH, flat=True)
 
     assert True
+
+def test_restore_container_autorestart(duthost):
+    command_output = duthost.shell("show feature autorestart", module_ignore_errors=True)
+    if command_output['rc'] != 0:
+        logging.info("Feature autorestart utility not supported. Error: {}".format(command_output['stderr']))
+        logging.info("Skipping disable_container_autorestart")
+        return
```

Comment:
We can get rid of this check since this will always be used after pretest. In case the cmd is not supported, the file will not exist and we will just return

Reply:
Updated. Thanks